src/Symfony/Component/Security/Http/AccessToken/Oidc/OidcTokenHandler.php

```diff
@@ -37,10 +41,13 @@
 final class OidcTokenHandler implements AccessTokenHandlerInterface
 {
     use OidcTrait;
+    private ?JWKSet $decryptionKeyset = null;
+    private ?AlgorithmManager $decryptionAlgorithms = null;
+    private bool $enforceEncryption = false;
 
     public function __construct(
         private Algorithm|AlgorithmManager $signatureAlgorithm,
-        private JWK|JWKSet $jwkset,
+        private JWK|JWKSet $signatureKeyset,
```

Comment:
I'm wondering if changing the argument/property name may cause a BC break, particularly about named parameters.

Reply:
AFAIK, there is no BC promise regarding the argument names (except for attributes).
@nicolas-grekas: do you confirm? Is it possible to change the argument names here?

Reply:
See `[10]` of https://symfony.com/doc/current/contributing/code/bc.html